modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/RelatedObjectEntryResourceTest.java

```diff
@@ -107,47 +108,203 @@ public void setUp() throws Exception {
 			_systemObjectDefinitionMetadataTracker.
 				getSystemObjectDefinitionMetadata("User");
 
-		ObjectDefinition userSystemObjectDefinition =
+		_userSystemObjectDefinition =
 			_objectDefinitionLocalService.fetchSystemObjectDefinition(
 				_userSystemObjectDefinitionMetadata.getName());
 
-		_objectRelationship =
-			ObjectRelationshipLocalServiceUtil.addObjectRelationship(
-				TestPropsValues.getUserId(),
-				_objectDefinition.getObjectDefinitionId(),
-				userSystemObjectDefinition.getObjectDefinitionId(), 0,
-				ObjectRelationshipConstants.DELETION_TYPE_PREVENT,
-				LocalizedMapUtil.getLocalizedMap(RandomTestUtil.randomString()),
-				StringUtil.randomId(),
-				ObjectRelationshipConstants.TYPE_MANY_TO_MANY);
-
 		_user = TestPropsValues.getUser();
-
-		ObjectRelationshipLocalServiceUtil.
-			addObjectRelationshipMappingTableValues(
-				TestPropsValues.getUserId(),
-				_objectRelationship.getObjectRelationshipId(),
-				_objectEntry.getPrimaryKey(), _user.getUserId(),
-				ServiceContextTestUtil.getServiceContext());
 	}
 
 	@After
 	public void tearDown() throws Exception {
-		ObjectRelationshipLocalServiceUtil.
-			deleteObjectRelationshipMappingTableValues(
-				_objectRelationship.getObjectRelationshipId(),
-				_objectEntry.getPrimaryKey(), _user.getUserId());
-		ObjectRelationshipLocalServiceUtil.deleteObjectRelationship(
-			_objectRelationship);
+		for (ObjectRelationship objectRelationship : _objectRelationships) {
```

Comment:
~~If you only create one objectRelationship in the setUp, you will not have to iterate through the relationships to delete all the data.~~
Nevermind